src/components/Page.jsx

```diff
@@ -27,6 +33,18 @@ const Page = () => {
     },
   ];
 
+  const validateFormData = ({activeStep}) => {
+
+    // maybe put a switch statement here to do a different validation for each card?
+    // won't be able to find by Id from another card
+    // should maybe also move the validations to its own component
+    let username = document.getElementById("name").value;
```

Comment:
don't look up via the document. You can check the active step and then check if each field is not empty biased on the step. The problem with my suggestion is order matters.

Reply:
shouldn't you have an on change on the PersonalInfo step to populate values? 

where is the onChange here?
```
<input type="text" id="name" placeholder="e.g. Stephen King" required />
```